### ISSUE TITLE
⌨ Accessibility - Make items in My Skills list heading level 3 (instead of 2)

### ISSUE THREAD

**Issue by MaddyDaigle** [GCTC-NTGC/gc-digital-talent#5008]
## ❌ Current behaviour (issue)
Currently, the headers under "My Skills and Experience" for each type of experience (in the tabs) are heading level 2.  Since they are logically under "My Skills and Experience" which is a heading level 2, they should be a level 3 instead.

<img width="1056" alt="Screen Shot 2022-12-02 at 9 43 16 PM" src="https://user-images.githubusercontent.com/1794660/205418790-d5444e07-c675-43f5-8a99-21fe379afc28.png">

**WCAG 2.1 Criteria:**

### How to reproduce
1. Environment (prod or UA?) Prod
2. Which browser and/or Assistive Technology? Chrome
3. Step-by-step instructions explaining how to reproduce the problem.

## ✔ Expected behaviour (fix)
Change the `<h2>`s for each header in the tabs under the "My Skills and Experience" to `<h3>`s.  See the elements outlined in red in the above image.

### Acceptance Criteria
A set of assumptions which, when tested, verify that the accessibility requirement was met.
- [ ] Criteria
- [ ] Criteria 2

## Additional information (optional)
1. How does it impact users?
2. Related issues, suggestions how to fix
